go/common/enclave.go

```diff
@@ -42,26 +43,26 @@ type Enclave interface {
 	SubmitL1Block(block L1Block, receipts L1Receipts, isLatest bool) (*BlockSubmissionResponse, error)
 
 	// SubmitTx - user transactions
-	SubmitTx(tx EncryptedTx) (EncryptedResponseSendRawTx, error)
+	SubmitTx(tx EncryptedTx) responses.SendRawTx
```

Comment:
I'd get rid of the verb in front of these responses. It applies to all of them.
"RawTx", "TxCount", etc